### PR TITLE
Update sms and email confirmation templates

### PR DESCRIPTION
**Pull request by JamesGardiner**
### What is the context of this PR?
Adds appropriate panel calls to email and sms confirmation pages

### How to review 
Check that the new content matches that of the corresponding trello card

### Checklist

* [x] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by JamesGardiner** [comment]
https://trello.com/c/ihCQC8Dc/4196-content-addition-for-text-message-and-email-receipts-xs

File: templates/confirmation-email-sent.html [hunk 1]
```diff
@@ -21,4 +21,13 @@ <h1 data-qa="confirmation-text">{{ _("A confirmation email has been sent to {ema
 
   {% include 'partials/sign-out-pre-footer-panel.html' %}
 
+  {% call onsPanel({
```
Comment (LJBabbage) [inline templates/confirmation-email-sent.html:24]: The design has it before "Didn't receive an email?"

**Review by LJBabbage — CHANGES_REQUESTED** [review]
Works fine, just one in a different position than the design

**Comment by JamesGardiner** [comment]
That's what I get for pushing code at 10pm... Will update this afternoon 👍

File: templates/confirmation-email-sent.html (hunk 1 above)
Comment (JamesGardiner) [reply templates/confirmation-email-sent.html:17]: Done 👍

**Review by petechd — APPROVED** [review]
Tested locally, it looks correct now after you moved it around.

**Review by LJBabbage — APPROVED** [review]

**Pull request merged**